crypto/bindings/conversion.ts

```diff
@@ -47,18 +64,59 @@ function createRustConversion(wasm: wasm) {
         let rustCoeffs = fieldsToRustFlat(coeffs);
         return new WasmGate(typ, rustWires, rustCoeffs);
       },
+      pointToRust(point: OrInfinity) {
+        return affineToRust(point, makeCommitmentCurve);
+      },
+      pointFromRust(point: WasmAffine) {
+        return affineFromRust(point);
+      },
+      pointsToRust(points: MlArray<OrInfinity>) {
+        return mlArrayToRustVector(points, affineToRust, makeCommitmentCurve);
+      },
+      pointsFromRust(points: Uint32Array) {
+        return mlArrayFromRustVector(
+          points,
+          CommitmentCurve,
+          affineFromRust,
+          false
+        );
+      },
+      polyCommToRust(polyComm: PolyComm): WasmPolyComm {
+        return polyCommToRust(polyComm, WasmPolyComm, makeCommitmentCurve);
+      },
+      polyCommFromRust(polyComm: WasmPolyComm): PolyComm {
+        return polyCommFromRust(polyComm, CommitmentCurve, false);
+      },
     };
   }
 
-  const fpConversion = perField(wasm.WasmFpGate);
-  const fqConversion = perField(wasm.WasmFqGate);
+  // TODO: we have to lie about types here:
+  // -) the WasmGVesta class doesn't declare __wrap() but our code assumes it
```

Comment:
we need to use this so that we can instantiate these classes from pointers. It's sad that the autogenerated Wasm types don't reflect the full functionality, but there are no bad consequences other than having to coerce the type rather than use the declared type, in this one line

Reply:
Does it come from `wasm-bindgen` not generating `__wrap()` in ts types? :thinking:

Reply:
Yes